invenio_vocabularies/datastreams/readers.py

```diff
@@ -116,6 +117,32 @@ def read(self, item=None, *args, **kwargs):
         yield from self._iter(url=url, *args, **kwargs)
 
 
+class RORHTTPReader(BaseReader):
```

Comment:
I think putting this under datastreams makes sense, though it's more specific than what's been in datastreams in the past. Not for this PR, but this would indicate to me that `RORTransformer` should be moved to `datastreams/transformers.py`

Reply:
I put it there since it is going to be used for affiliations too, but I agree about it being more specific than other common readers.

---

Concerning the proposed alternatives:
* `contrib/ror/datastreams.py`: It might make people think that `ror` is a vocabulary since all other modules in `contrib` are.
* `contrib/common/...`: It makes it more clear that `common` is not a vocabulary, but it might end up containing a lot of unrelated readers/transformers/writers.

Any preference?

---

Note that if we create a new `VOCABULARIES_DATASTREAM_READER` we'll then need to add it in [`invenio-app-rdm`'s config](https://github.com/inveniosoftware/invenio-app-rdm/blob/master/invenio_app_rdm/config.py#L650).

Reply:
I agree `/contrib/ror' might get confusing. I'd say`/contrib/common/ror.py` or put them in `/datastreams`